lisa/schema.py

```diff
@@ -504,6 +504,87 @@ def _generate_min_capability(self, capability: Any) -> Any:
         return min_value
 
 
+@dataclass_json()
+@dataclass()
+class NetworkInterfaceOptionSettings(FeatureSettings):
+    type: str = "NetworkInterface"
+    enable_sriov: Optional[bool] = None
```

Comment:
I thought it again. We need a SetSpace, it should include Synthetic, Sriov. So it mean what the network interface can be. In Azure, all network interface support Synthetic, but some of them support Sriov. It means all network interface can run test cases of Synthetic. If the Synthetic is selected, the deployment should make sure the network interface is deployed as synthetic, even it support Sriov. But if nothing specified, the sriov should be enabled by default, if the network interface supports it.